app/code/Magento/Checkout/Test/Mftf/Test/StorefrontCheckoutVerifyAuthenticatePopupNotVisibleTest.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontCheckoutVerifyAuthenticatePopupNotVisibleTest">
+        <annotations>
+            <features value="Checkout"/>
+            <stories value="Authentication popup should be removed from Checkout page"/>
+            <title value="Verify authenticate popup not visible in checkout page"/>
+            <description value="Verify authenticate popup not visible in checkout page"/>
+            <severity value="AVERAGE"/>
+            <testCaseId value="MC-36218"/>
+            <group value="checkout"/>
+        </annotations>
+
+        <before>
+            <!--Create Simple Products-->
+            <createData entity="SimpleSubCategory" stepKey="createCategory"/>
+            <createData entity="SimpleProduct" stepKey="createSimpleProduct">
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
+        </before>
+        <after>
+            <!--Delete created data-->
+            <deleteData createDataKey="createCategory" stepKey="deleteCategory"/>
+            <deleteData createDataKey="createSimpleProduct" stepKey="deleteSimpleProduct"/>
+        </after>
+
+        <!--Go to product page-->
+        <actionGroup ref="StorefrontOpenProductPageActionGroup" stepKey="openStorefrontProductPage">
+            <argument name="productUrl" value="$$createProduct.custom_attributes[url_key]$$"/>
```

Comment:
I believe the possible reason for that -  we use `$$createProduct` but in the "before" section we use `createSimpleProduct` as a key.